examples/flutter_gallery/test/demo/material/drawer_demo_test.dart

```diff
@@ -21,7 +21,7 @@ void main() {
     final double initialTopItemSaneY = tester.getTopLeft(find.text('Drawer item A')).dy;
     expect(initialTopItemSaneY, greaterThan(0.0));
 
-    await tester.drag(find.text('Drawer item B'), const Offset(0.0, 400.0));
+    await tester.drag(find.text('Drawer item B'), const Offset(0, 400));
     await tester.pump();
 
     expect(tester.getTopLeft(find.byType(UserAccountsDrawerHeader)).dy, 0.0);
```

Comment:
And here?

Reply:
There is no context type here, the second argument of `expect` has type `dynamic`. There is no way to guess whether changing 0.0 to 0 is correct, short of recognizing `package:test`'s `expect` function and knowing how it uses its second argument if it is a number value. (It uses it as the first operand of an `==` operator, if it had been the second operand, the change would just be unsafe).